### PR TITLE
Separating user v core api calls

### PR DESCRIPTION
**Pull request by vimukthi-git**
<This is intended as a guide when describing-reviewing pull requests for go-centrifuge>
Issue Link: #1142 
Screenshot 2019-07-01 at 10 46 56

### CODE REVIEW

File: documents/service.go [hunk 1]
```diff
@@ -273,8 +273,10 @@ func (s service) ReceiveAnchoredDocument(ctx context.Context, model Model, colla
 		DocumentId:   hexutil.Encode(model.ID()),
 	}
 
-	// Async until we add queuing
-	go s.notifier.Send(ctx, notificationMsg)
+	status, err := s.notifier.Send(ctx, notificationMsg)
```
Comment (vimukthi-git) [inline documents/service.go]: bugfix

**Comment by vimukthi-git** [comment]
@charleenfei @vedhavyas closing this as I don't have any idea about the design discussions you had. I only know what was explained to me by @mikiquantum. Please make sure to document your discussions/designs in the future, since you never know when other team members would have to take over when you are not here. It's clear to me this pr is off from the goal @ved had in mind. Therefore he would take over. Thanks :)

**Comment by ved** [comment]
What is it regarding?

**Comment by ved** [comment]
Please stop this

**Pull request closed without merging**